opentelemetry/src/logs/record.rs

```diff
@@ -331,10 +331,10 @@ impl LogRecordBuilder {
     }
 
     /// Assign body
-    pub fn with_body(self, body: AnyValue) -> Self {
+    pub fn with_body(self, body: impl Into<AnyValue>) -> Self {
         Self {
             record: LogRecord {
-                body: Some(body),
+                body: Some(body.into()),
```

Comment:
nit - can you also change the code [here](https://github.com/open-telemetry/opentelemetry-rust/blob/7d0b80ea852eb3218504b722476484063802a9a4/opentelemetry-sdk/benches/log.rs#L60) and few other places not to explicitly use `into`.